sklearn/ensemble/bagging.py

```diff
@@ -101,7 +101,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
 
                 curr_sample_weight[not_indices] = 0
 
-            estimator.fit(X[:, features], y, sample_weight=curr_sample_weight)
+            if sample_weight is None:
+                estimator.fit(X[:, features], y)
+            else:
+                estimator.fit(X[:, features], y, sample_weight=curr_sample_weight)
```

Comment:
Temporary: I will re-implement the changes in 9d3becf and also comment out the tests with `LogisticRegression` in `test_sample_weight_missing` of test_weight_boosting.py  - to see if there are any other issues beyond these two.